cli/src/logger/mod.rs

```diff
@@ -1,21 +1,12 @@
-use log::SetLoggerError;
-
-mod simple;
-pub use self::simple::SimpleLogger;
-
-mod sematext;
-pub use self::sematext::Sematext;
-
-mod logzio;
-pub use self::logzio::LogzIO;
-
-pub fn init() -> Result<(), SetLoggerError> {
-  let env = std::env::var("ALAN_TECH_ENV").unwrap_or("production".to_string());
-  let simple_logger = Box::new(SimpleLogger);
-  let sematext = Box::new(Sematext);
-  let logzio = Box::new(LogzIO);
-  match env.as_str() {
-    "local" => multi_log::MultiLogger::init(vec![simple_logger], log::Level::Info),
-    _ => multi_log::MultiLogger::init(vec![sematext, logzio], log::Level::Info),
-  }
+#[macro_export]
+macro_rules! error {
+  ($errName:tt, $($message:tt)+) => {async{
+      eprintln!($($message)+);
+      client_error($errName).await;
+  }};
+  (metadata: $metadata:tt, $errName:tt, $($message:tt)+) => {async{
+    let value = json!($metadata);
+    eprintln!($($message)+);
+    client_error($errName).await;
+  }}
```

Comment:
maybe we could have a 3rd parameter which is an optional status code and when it is present then we call `std::process::exit` with it?

Reply:
Hmm not sure if we should add that kind of logic here, in cases like [this](https://github.com/alantech/anycloud/pull/44/files#diff-88709fe0996d31ad67c84bd6a39f8e3fedb412156d16fb220b2d1921bd632861R154-R159) will be complicated to add hints or extra info after the action if needed before the exit.

Reply:
My thought was that in cases where we still print a hint after, we set it to `None` in which case it doesn't exit within the macro. It would be an `Option<u64>`.

Reply:
So far could not managed to define an optional arg like that, what I've achieved is something like the following:
```rust
#[macro_export]
macro_rules! error {
  ($errName:tt, $($message:tt)+) => {async{
      eprintln!($($message)+);
      client_error($errName).await;
  }};
  (metadata: $metadata:tt, $errName:tt, $($message:tt)+) => {async{
    let value = json!($metadata);
    eprintln!($($message)+);
    client_error($errName).await;
  }};
  (exit: $exit:tt, $errName:tt, $($message:tt)+) => {async{
    eprintln!($($message)+);
    client_error($errName).await;
    if $exit {
      std::process::exit(1);
    }
  }}
}
```
And the call will be something like `error!(exit: true, "TEST_ERR", "test").await;`

Also, in this case as you see wont be able to add metadata in case it's necessary.

Reply:
Ah in that case let's just leave it as-is!